testsuite/features/step_definitions/common_steps.rb

```diff
@@ -222,7 +222,7 @@
 end
 
 Then(/^I should see the CPU frequency of the client$/) do
-  get_cpu_freq = "lscpu  | grep 'CPU MHz'" # | awk '{print $4}'"
+  get_cpu_freq = "cat /proc/cpuinfo  | grep 'cpu MHz'" # | awk '{print $4}'"
```

Comment:
You can add here `grep -i ` to make it agnostic for capitals or small, but just a suggestion, I assume this is the current output.